### PR TITLE
New website page (training) and subpage (Orientation to RStudio)

### PR DESCRIPTION
**Pull request by nsbatra**
Work-in-progress, mostly to test pushing.

### CODE REVIEW

File: content/training/RStudio_overview.Rmd [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Open RStudio. For RStudio to function you must also have R installed on the computer (see the X page for installation instructions). RStudio is an interface (GUI) for easier use of R. You can think of R as being the engine of a vehicle, doing the crucial work, and RStudio as the body of the vehicle (with seats, accessories, etc.) that helps you actually use the engine to move forward!
+
+By default RStudio displays four rectangle panes. 
+
+![](/images/RStudio_overview.png)
+*If your RStudio displays only one left pane it is because you have no scripts open yet. See below how to open or start a new script.*
+
+
+
+##The R Console Pane
```
Comment (zkamvar) [inline content/training/RStudio_overview.Rmd]: Please give the markdown some room to breathe. hashes and text should always be separated by a space  and there should be a newline after section headers.

**Comment by zkamvar** [comment]
The R4EPIs-website2.Rproj file is not necessary. Please remove it.

File: content/training/RStudio_overview.Rmd (hunk 1 above)
Comment (nsbatra) [reply content/training/RStudio_overview.Rmd]: I added the spaces and newlines - please let me know if there places you were referring to that I didn't address

Comment (zkamvar) [reply content/training/RStudio_overview.Rmd]: That wasn't *exactly* what I was addressing. When I said to add newlines, I meant actual newlines:

so go from this

```
##Section header
text text text

##Section two
text text text
```

to this


```
## Section header

text text text

## Section two

text text text
```

Comment (zkamvar) [reply content/training/RStudio_overview.Rmd]: The slashes you inserted will not do anything. You might be thinking of LaTeX.

**Comment by zkamvar** [comment]
Just remember to let us know when it's ready for review.

**Comment by zkamvar** [comment]
Hi @nsbatra,

It looks like #2 includes the commits from this PR. Should we delete this PR and focus on #2?

**Comment by nsbatra** [comment]
Closing because the commits are part of the R Basics branch (which was a mistake). This will simplify the review.

**Pull request closed without merging**